README.md

```diff
@@ -1 +1,76 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
```

Comment:
Great job @Donmark2k 🤝

- [ ] Kindly update all sections of your readme file. Right now you only have a template without your own content. Kindly update the project name, description, tools used, social handles, etc.